components/automate-deployment/pkg/preflight/checks.go

```diff
@@ -33,8 +35,19 @@ func RootUserRequiredCheck() Check {
 	}
 }
 
-func DefaultMinimumDiskCheck() Check {
-	return MinimumDiskCheck(5 * (1 << 30))
+const GB = 1 << 30
+
+func minDiskBytesForConfig(c *dc.AutomateConfig) uint64 {
+	collections := deployment.CollectionsForConfig(c.GetDeployment())
+	if stringutils.SliceContains(collections, services.BuilderCollectionName) {
+		return 15 * GB
```

Comment:
[nit] should we sum across the products. So builder alone with be 15. builder + automate would be 20